update-version.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# Change version
+yarn run lerna version $1 --ignore-changes --skip-git
+
+# prepare packages
+yarn
+yarn workspace @datadog/mobile-react-native pack
+yarn workspace @datadog/mobile-react-navigation pack
+yarn workspace @datadog/mobile-react-native-navigation pack
+
+./check-release-content.sh -p packages/core/datadog-mobile-react-native-v$1.tgz > packages/core/release-content.txt
+./check-release-content.sh -p packages/react-navigation/datadog-mobile-react-navigation-v$1.tgz > packages/react-navigation/release-content.txt
+./check-release-content.sh -p packages/react-native-navigation/datadog-mobile-react-native-navigation-v$1.tgz > packages/react-native-navigation/release-content.txt
+
+
+# Update example repo
+(cd example &&
+yarn remove @datadog/mobile-react-native &&
+yarn remove @datadog/mobile-react-navigation &&
+yarn remove @datadog/mobile-react-native-navigation &&
+yarn add file:../packages/core
+yarn add file:../packages/react-native-navigation
+yarn add file:../packages/react-navigation
+(cd ios && pod install)
```

Comment:
I guess at least on the local machine `pod install` will fail with version mismatch error and we need to use `pod update` (by looking on the description from [here](https://guides.cocoapods.org/using/pod-install-vs-update.html)), considering pods were already installed before, but not quite sure about that.

Reply:
So actually we need to do the `pod update` when we increase the bridge SDK version, but that should be done in another PR so it won't be needed when we run this script.
Still I'ev added --repo-update to the pod install to make sure we fetch the new versions if this has not been done before